docs/docs/widgets/button.md

```diff
@@ -3,3 +3,35 @@ sidebar_position: 1
 ---
 
 # Button
+
+Button widget can be used for allowing the users to take actions with a single click.
+
+<img class="screenshot-full" src="/img/widgets/button/adding-button.gif" alt="ToolJet - Widget Reference - Button" height="420"/>
+
+## Event: On click
+
+On click event is triggered when button is clicked and sets the action to run.
```

Comment:
This event is triggered when the button is clicked. Just like any other event on ToolJet, you can set multiple handlers for `on click` event.